src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+                        <>
+                            <ContainerHeder style={{ padding: '0 32px' }}>
+                                <LeftBlock>
+                                    <LogoHead />
+                                    <PointBalUser />
+                                </LeftBlock>
+                                <MenuHeaderBlock>
+                                    <MobileHamburgerContainer onClick={toggle} >
+                                        <MobileHamburgerLine />
+                                    </MobileHamburgerContainer>
+                                    <MenuInitiaLetterCircle style={{ margin_left: '32px' }}>
+                                        <MenuInitiaLetter>{nameLetter}</MenuInitiaLetter>
+                                    </MenuInitiaLetterCircle>
+                                    <MenuName>{UserName}</MenuName>
+                                    <NavLink to="login" onClick={`${toggle}${logOut}`}>
+                                        <ExitIcon />
+                                    </NavLink>
+                                </MenuHeaderBlock>
+                            </ContainerHeder>
+                            <MobileHamburgerbackground style={{ transform: isOpen ? "translateX(0%)" : "translateX(120%)", opacity: isOpen ? "1" : "0.3" }}>
+                                <MobileHamburgerCrossBox onClick={toggle}>
+                                    <MobileHamburgerCross>
+                                        <MobileHamburgerCross1 />
+                                    </MobileHamburgerCross>
+                                </MobileHamburgerCrossBox>
+                                <NavLink to="home" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenuUp>Главная</MobileHamburgerSectionMenuUp>
+                                </NavLink>
+                                <NavLink to="planning" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Планирование</MobileHamburgerSectionMenu>
+                                </NavLink>
+                                <NavLink to="awards" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Награды</MobileHamburgerSectionMenu>
+                                </NavLink>
+                                <NavLink to="contacts" onClick={toggle}
+                                    style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                    <MobileHamburgerSectionMenu>Контакты</MobileHamburgerSectionMenu>
+                                </NavLink>
```

Comment:
this code looks too repetitive, I think React gives us a possibility to elegantly lower code lines amount using Array.map method

Reply:
Thanks for the advice) I'll try to change the approach using the map.